### PR TITLE
Added example

### PR DESCRIPTION
**Pull request by rl-saen**
Added an example on how to retrieve the uid of a copied record.

### CODE REVIEW

File: Documentation/ApiOverview/Typo3CoreEngine/Database/Index.rst [hunk 1]
```diff
@@ -317,7 +317,21 @@ Examples of commands:
 .. index:: DataHandler; Data array
 .. _tce-data:
 
-Data array
+Accessing the uid of copied records:
+------------------------------------
+
```
Comment (mbrodala) [inline Documentation/ApiOverview/Typo3CoreEngine/Database/Index.rst:322]: I'd first start with an explanation of `copyMappingArray_merged` here, when it is filled, how it's structure looks like, etc. Then the following example is way easier to understand then just :man_shrugging:

File: Documentation/ApiOverview/Typo3CoreEngine/Database/Index.rst [hunk 2]
```diff
@@ -317,7 +317,21 @@ Examples of commands:
...
+::
+
+   $cmd['tt_content'][1203]['copy'] = 400;  // Copies tt_content uid=1203 to first position in page uid=400
+   $dataHandler->start([], $cmd);
+   $dataHandler->process-cmdmap()
+   
+   $uid = $dataHandler->copyMappingArray_merged['tt_content][1203];
+
+
+..
```
Comment (mbrodala) [inline Documentation/ApiOverview/Typo3CoreEngine/Database/Index.rst:332]: This and the empty lines before should be dropped.

**Review by mbrodala — CHANGES_REQUESTED** [review]

File: Documentation/ApiOverview/Typo3CoreEngine/Database/Index.rst (hunk 1 above)
Comment (rl-saen) [reply Documentation/ApiOverview/Typo3CoreEngine/Database/Index.rst:322]: Well, it's pretty much 🤷‍♂️ to me as well, as I just stumbled upon it. 😄 

I added my observations  and a warning about `copyMappingArray`.

File: Documentation/ApiOverview/Typo3CoreEngine/Database/Index.rst [hunk 3]
```diff
@@ -317,7 +317,39 @@ Examples of commands:
 .. index:: DataHandler; Data array
 .. _tce-data:
 
-Data array
+Accessing the uid of copied records:
+------------------------------------
+
+Every instance of DataHandler contains an array called :php:`$copyMappingArray_merged` which holds information about all copy operations the instance has performed.
```
Comment (mbrodala) [inline Documentation/ApiOverview/Typo3CoreEngine/Database/Index.rst:323]:
```suggestion
The :php:`DataHandler` keeps track of records created by :code:`copy` operations in its :php:`$copyMappingArray_merged` property. This property is public and can be used to determine the UID of a record copy based on the UID of the copied record.
```

File: Documentation/ApiOverview/Typo3CoreEngine/Database/Index.rst [hunk 4]
```diff
@@ -317,7 +317,39 @@ Examples of commands:
...
+
+.. caution::
+   Not to be confused with :php:`$copyMappingArray` which contains only information about the last copy operation and is cleared between new ones.
```
Comment (mbrodala) [inline Documentation/ApiOverview/Typo3CoreEngine/Database/Index.rst:326]:
```suggestion
   The :php:`$copyMappingArray_merged` property should not be confused with the :php:`$copyMappingArray` property which contains only information about the last copy operation and is cleared between each operation.
```

File: Documentation/ApiOverview/Typo3CoreEngine/Database/Index.rst [hunk 5]
```diff
@@ -317,7 +317,39 @@ Examples of commands:
...
+
+::
```
Comment (mbrodala) [inline Documentation/ApiOverview/Typo3CoreEngine/Database/Index.rst:328]:
```suggestion
The structure of the :php:`$copyMappingArray_merged` property looks like this::
```

File: Documentation/ApiOverview/Typo3CoreEngine/Database/Index.rst [hunk 6]
```diff
@@ -317,7 +317,39 @@ Examples of commands:
...
+
+   $copyMappingArray = [
```
Comment (mbrodala) [inline Documentation/ApiOverview/Typo3CoreEngine/Database/Index.rst:330]:
```suggestion
   $copyMappingArray_merged = [
```

File: Documentation/ApiOverview/Typo3CoreEngine/Database/Index.rst [hunk 7]
```diff
@@ -317,7 +317,39 @@ Examples of commands:
...
+      'tt_content' => [
+         1203 => 1275,
+         54 => 1276
+      ],
+      'pages' => [
+         891 => 1277
+      ]
```
Comment (mbrodala) [inline Documentation/ApiOverview/Typo3CoreEngine/Database/Index.rst:337]:
```suggestion
      <table> => [
         <original-record-uid> => <record-copy-uid>,
      ],
```

File: Documentation/ApiOverview/Typo3CoreEngine/Database/Index.rst [hunk 8]
```diff
@@ -317,7 +317,39 @@ Examples of commands:
...
+   ];
+..
+
+The keys of the array represent the names of the manipulated tables which in return contain arrays with the uids of original records as keys. The values of each index in the innermost arrays represent the newly copied records uid.
```
Comment (mbrodala) [inline Documentation/ApiOverview/Typo3CoreEngine/Database/Index.rst:341]:
```suggestion
The property contains the names of the manipulated tables as keys and a map of original record UIDs and UIDs of record copies as values.
```

**Review by mbrodala — CHANGES_REQUESTED** [review]

**Review by mbrodala — APPROVED** [review]

**Comment by mbrodala** [comment]
@rl-saen Thanks for the suggestion and keeping up with the requested changes. ;-)

**Pull request merged**